src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/MediationJobSchedulerService.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.cmc.claimstore.jobs;
+
+import org.quartz.JobDetail;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.Trigger;
+import org.quartz.CronScheduleBuilder;
+import org.quartz.impl.StdSchedulerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.MediationEmailJob;
+
+import static org.quartz.JobBuilder.newJob;
+import static org.quartz.TriggerBuilder.newTrigger;
+
+@Component
+public class MediationJobSchedulerService {
+
+    public void scheduleMediation() {
+
+        try {
+            Scheduler scheduler = StdSchedulerFactory.getDefaultScheduler();
```

Comment:
You don't need to start/stop scheduler explicitly. I hope you want to use the same scheduler instance we are using already. Please have a look at `JobSchedulerService` for reference

Reply:
I saw that, i thought that was used for different purpose...i didn't spend too much time on it yet. I will look in to that when i get back to this task..